.github/workflows/verify.yml

```diff
@@ -33,20 +33,13 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - uses: actions/setup-ruby@v1
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
-
-      - name: Setup bundler
-        run: |
-          gem install bundler
-
-      - name: Bundle install
-        run: |
-          bundle config path vendor/bundle
-          bundle install --jobs 4 --retry 3
+          bundler-cache: true
         env:
-          BUNDLER_WITHOUT: coverage development pcap
+          BUNDLE_WITHOUT: "coverage pcap"
```

Comment:
Why the drop of `development` ? 👀

Reply:
`BUNDLER_WITHOUT: coverage development pcap` would make the tests fail with the error `metasploit-yard not in bundle, so can't setup yard tasks. To run yard ensure to install the development group.`. Dropping `development` fixes that and makes the tests complete.

Reply:
Looks like we'd need to split out the gemfile to specify some of the development dependencies as being test dependencies. Happy to punt on this.